eo-maven-plugin/src/test/java/org/eolang/maven/RegisterMojoTest.java

```diff
@@ -91,4 +91,37 @@ void doesNotFailWhenNoStrictNames(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void registersInExternal(@TempDir final Path temp) throws IOException {
+        new Home(temp).save(
+            new ResourceOf("org/eolang/maven/file-name/abc-def.eo"),
+            Paths.get("src/eo/org/eolang/maven/foo.eo")
+        );
+        final String name = "org.eolang.maven.foo";
+        final String source = "src/eo";
+        final FakeMaven maven = new FakeMaven(temp)
+            .with("sourcesDir", temp.resolve(source).toFile())
+            .execute(new FakeMaven.Register());
+        MatcherAssert.assertThat(
+            String.format(
+                String.join(
+                    " ",
+                "Source object %s placed in %s should have been registered in external tojos",
+                    "but it have not"
+                ),
+                name,
+                source
+            ),
+            maven.external()
+                .getById(name)
+                .exists("id"),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            "External and foreign tojos should have the same status after registering",
```

Comment:
@maxonfjvipon Could you please, add more info here? "Why should they have the same status"?